configure.py

```diff
@@ -346,6 +346,11 @@
     dest='trace_maps',
     help='Enable the --trace-maps flag in V8 (use at your own risk)')
 
+parser.add_option('--enable-pointer-compression',
```

Comment:
Perhaps change to `--experimental-pointer-compression` for consistency with other similar flags?

Reply:
Updated, please review.